webcompat/static/js/lib/dashboard.js

```diff
@@ -47,6 +47,27 @@ const filteringSort = issuesNumber => {
   });
 };
 
+/* viewMode */
+const viewMode = () => {
+  const view = document.getElementById("js-Filter-view").value;
+  const Triages = document.getElementById("js-Triages");
```

Comment:
Most JS style recommends not using Uppercase except for classes/prototypes. Is Uppercase intentional here?

(it doesn't matter, just curious)

Reply:
Ooups no, it’s not.